scr/views/article.ejs

```diff
@@ -0,0 +1,44 @@
+<style>
+  .article_img{
+    width:100%;
+    max-width:900px;
+    height: auto;
+    margin-top: 15px;
+    }
+
+    .article-container{
+      margin-top: 30px;
+    }
+</style>
+
+<%- include('./partials/head'); %>
+<body>
+
+  <header>
+    <%- include('./partials/header'); %>
+  </header>
+
+  <div class="container article-container">
+    <div class="row">
+      <div class="images col-sm-12 col-md-8 col-lg-8 col-xl-8">
+        <a class="back" href="/api/products">Back</a>
```

Comment:
As a general comment for all the static texts in the view need to be localized, for example here: `Back`